Assets/Scripts/Controllers/CharacterSpriteController.cs

```diff
@@ -49,8 +49,9 @@ public void OnCharacterCreated(Character c)
         char_go.transform.SetParent(this.transform, true);
 
         SpriteRenderer sr = char_go.AddComponent<SpriteRenderer>();
-        sr.sprite = SpriteManager.current.GetSprite("Character", "p1_front");
+		sr.sprite = SpriteManager.current.GetSprite("Character", "p2_front");
```

Comment:
Old code:
![screen snippet one](http://puu.sh/qHcSx/f2aa3a3489.png)

New code:
![screen snippet two](http://puu.sh/qHcWW/1ea79a6800.png)

I have marked what the indentations are made of in each snippet. Your tabs aren't turned into four spaces.